ui/src/components/LandingPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import OverviewTable from './OverviewTable';
+
+class LandingPage extends React.Component {
+  constructor () {
+    super();
+    this.state = {};
+  }
+  render () {
+    return (
+      <div className="landingPage">
```

Comment:
Is this wrapping div a requirement in react?

Reply:
Not required no, can remove it.